app/views/medications/new.html.erb

```diff
@@ -1,8 +1,11 @@
 <p>Enter brand medication name</p>
 <p style="color:red"><%= render partial: '/partials/flash' %></p>
 <section class="med_search_form">
-  <%= form_tag "/medications/search", method: :get %>
-  <%= label_tag :brand_name %>
-  <%= text_field_tag :brand_name %>
-  <%= submit_tag "Find Medication" %>
+  <%= form_tag "/medications/search", method: :get do %>
+    <%= label_tag :brand_name, 'Brand Name' %>
+    <%= text_field_tag :brand_name %>
+    <%= submit_tag "Find Medication" %>
+  <% end %>
 </section>
+<br>
+<%= button_to 'Back to Dashboard', '/dashboard', method: :get %>
```

Comment:
ahh this was my bad for not being more explicit in the card: I meant a back to the search view not back to the dashboard

since there's a dashboard button in the nav bar having a dashboard button here is redundant. I was thinking it would make it more user friendly though, when a user is on the search results page, for them to be able to go back to the page with the search bar in order to re-enter their medication search terms if they didn't get the search results they were looking for.